libres/lib/include/ert/enkf/local_obsdata_node.hpp

```diff
@@ -53,7 +57,7 @@ bool local_obsdata_node_all_timestep_active(
 bool local_obsdata_node_has_tstep(const local_obsdata_node_type *node,
                                   int tstep);
 void local_obsdata_node_reset_tstep_list(local_obsdata_node_type *node,
-                                         const int_vector_type *step_list);
+                                         const std::vector<int> step_list);
```

Comment:
```suggestion
                                         const std::vector<int>& step_list);
```
Also, making this `extern "C++"`.